docs/en/setup/service-agent/java-agent/Plugin-list.md

```diff
@@ -23,7 +23,7 @@
 - grpc-1.x
 - gson-2.8.x
 - h2-1.x
-- hbase-1.x
+- hbase-1.x/2.x
```

Comment:
Is this correct? Not `1.x-2.x`?

Reply:
Just like `tomcat-7.x/8.x` in the same file.